build.sh

```diff
@@ -369,16 +368,9 @@ function ensure_android_build {
         exit 1
     fi
 
-    local ndk_side_by_side="${ANDROID_HOME}/ndk/"
-    if [[ -d ${ndk_side_by_side} ]]; then
-        # shellcheck disable=SC2012
-        local ndk_version=$(ls "${ndk_side_by_side}" | sort -V | tail -n 1 | cut -f 1 -d ".")
-        if [[ ${ndk_version} -lt ${ANDROID_NDK_VERSION} ]]; then
-            echo "Error: Android NDK side-by-side version ${ANDROID_NDK_VERSION} or higher must be installed, exiting"
-            exit 1
-        fi
-    else
-        echo "Error: Android NDK side-by-side version ${ANDROID_NDK_VERSION} or higher must be installed, exiting"
+    # shellcheck disable=SC2012
+    if [[ -z $(ls "${ANDROID_HOME}/ndk/" | sort -V | grep "${FILAMENT_NDK_VERSION}") ]]; then
+        echo "Error: Android NDK side-by-side version ${FILAMENT_NDK_VERSION} or higher must be installed, exiting"
```

Comment:
The error message is incorrect now since the NDK version must match exactly.

Reply:
Not in all cases, but I'll adjust it to be "or compatible" instead of "or higher". Since FILAMENT_NDK_VERSION can be 20, 20.0, or 20.0.12312312313

Reply:
Ah good point, yeah that's nice. The error message just won't work with a fully specified version number but that's not a big deal. Although the problem with partial version numbers is that calls to sdkmanager in the other build.sh won't work (last I checked sdkmanager only accepts full version numbers now).

Reply:
> with partial version numbers is that calls to sdkmanager

true, but it's not expected that CI script will work with non-qualified versions